### ISSUE TITLE
Book 05: Sentence 035

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#8]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/587ab73a-2638-425a-b91c-74a7cec39375)

```
# sent_id = Book_05_-_The_Omens.txt:35
# text = Auh qujluja in maceoalli, in oqujmottiti: amo njmjtzcaoaz, ca onjmjtzan.
# text[norm] = Auh quilhuia in macehualli, in oquimottiti: amo nimitzcahuaz, ca onimitzan.
# text[orig] = Auh·qujluja·in·maceoalli·,·ino¶qujmottiti·:·amonjmjtzcaoaz·,·caonj¶mjtzan·.
# text[osp] = Y la persona, aqui en esta fantasma le auja aparecido, dezia la: no te dexare que ya te e caçado.
# text[spa] = Y dijo el vasallo, a quien se había hecho ver: No te dejaré, es que te he tomado.
# references = A&D/166:3
1	Auh	auh	CCONJ	_	_	2	cc	_	_
2	qujluja	ilhuia	VERB	_	_	0	root	_	_
3	in	in	DET	_	_	4	det	_	_
4	maceoalli	macehualli	NOUN	_	State=Abs	2	nsubj	_	_
5	,	,	PUNCT	_	_	7	punct	_	_
6	in	in	SCONJ	_	_	7	mark	_	_
7	oqujmottiti	mottitia	VERB	_	Aspect=Prior|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	4	acl	_	_
8	:	:	PUNCT	_	_	10	punct	_	_
9	amo	amo	ADV	_	_	10	advmod	_	_
10	njmjtzcaoaz	cahua	VERB	_	Number[obj]=Sing|Number[subj]=Sing|Person[obj]=2|Person[subj]=1|Subcat=Tran|Tense=Fut|VerbForm=Fin	2ccomp	_	_
11	,	,	PUNCT	_	_	13	punct	_	_
12	ca	cah|ca	VERB	_	_	13	_	_	_
13	onjmjtzan	ana	VERB	_	Aspect=Prior|Number[obj]=Sing|Number[subj]=Sing|Person[obj]=2|Person[subj]=1|Subcat=Tran|Tense=Past|VerbForm=Fin	10	_	_	_
14	.	.	PUNCT	_	_	2	punct	_	_
```

**Comment by ftyers** [comment]
Questions:
- Do we want `ccomp` or `parataxis` for reported speech? The docs are clear ([parataxis](https://universaldependencies.org/docs/u/dep/parataxis.html)), but rely on a standard function of the colon, which is not the case here. If we go with the docs, perhaps `parataxis:speech` would be a good idea.
- We have it appears two functions of *in* here, although in other places we only accept the `SCONJ` function. This needs to be checked and we need to come up with diagnostics. 
- What do we do with the *ca* + `VERB`? 

López Austín has:
> Y el hombre que lo había encontrado le decía: "No te dejo, que yo te he cautivado."

**Comment by ftyers** [comment]
Regarding *ca*, from Launey:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/ab32cd5b-51ab-4c39-b037-8d76f3b01d5e)

**Comment by ftyers** [comment]
How about this:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/bb0f6e77-563d-4bc7-b9dd-1145f2c94942)

Thoughts?

**Comment by Lguyogiro** [comment]
> Do we want ccomp or parataxis for reported speech? The docs are clear ([parataxis](https://universaldependencies.org/docs/u/dep/parataxis.html)), but rely on a standard function of the colon, which is not the case here. If we go with the docs, perhaps parataxis:speech would be a good idea.

Let's go with the docs, but yeah a subrelation like `speech` will be helpful, since I suspect we will end up with a ton of parataxis in this treebank.

**Comment by ftyers** [comment]
Is this one good to go?

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Fixed in 2e33f92.
